drivers/SmartThings/zwave-siren/profiles/aeon-siren.yml

```diff
@@ -10,6 +10,9 @@ components:
     version: 1
   categories:
   - name: Siren
+metadata:
+  mnmn: SmartThingsEdge
```

Comment:
Please use "SmartThings" instead of "SmartThingsEdge" - applies to all files in this PR